caffe2/python/onnx/backend.py

```diff
@@ -31,7 +31,7 @@
 import caffe2.python.utils
 import numpy as np
 import onnx
-from onnx import checker, GraphProto, TensorProto, AttributeProto, ModelProto
+from onnx import checker, GraphProto, TensorProto, AttributeProto, ModelProto, mapping
```

Comment:
Is `mapping` necessary? I didn't see it was used anywhere.

Reply:
Ouch

Reply:
Removed